handler/openid/helper.go

```diff
@@ -76,3 +88,27 @@ func (i *IDTokenHandleHelper) IssueExplicitIDToken(ctx context.Context, ar fosit
 	resp.SetExtra("id_token", token)
 	return nil
 }
+
+// ComputeHash computes the hash using the alg defined in the id_token header
+func (i *IDTokenHandleHelper) ComputeHash(ctx context.Context, sess Session, token string) (string, error) {
+	var err error
+	hash := sha256.New()
+	if alg, ok := sess.IDTokenHeaders().Get("alg").(string); ok {
+		if hashSize, err := strconv.Atoi(alg[2:]); err == nil {
```

Comment:
If `alg` is a string with `len() < 2` this will panic - can you please add a check to ensure this doesn't happen and also a test? The test can either be using Go fuzzing ( https://go.dev/doc/fuzz/ ) or be a hardcoded test.

Since this alg parameter can be user sent, it would be problematic that this can panic easily :)

Thanks!

Reply:
The IDTokenHelper is used in the handlers with the express intention of using it with JWTs. The consumer should accordingly use either a valid algorithm (RS256 etc.) or no alg in the ID token headers. I am happy to add the validation but I don't see this being a problem. For example - if someone uses `RSA256`or `R`, that would clearly be a bad implementation and easy for a developer to identify and fix.